### ISSUE TITLE
Ability to use cname in template

### ISSUE THREAD

**Issue by webvanced** [tuxis-ie/nsedit#150]
Hi,

It would be nice to be able to add a cname to the zonename in the template.
I made a small modification to the code to make this work. Would be nice to have this in the nsedit source.

Example template config:
` 
array(
            'name'      => '',
            'type'      => 'A',
            'content'   => '1.2.3.4'
		),
array(
            'name'      => '',
            'type'      => 'CNAME',
            'content'   => '[zonename]'
		)
`

In zones change add below 333 a line that replaces [zonenname].

```
            foreach ($template['records'] as $record) {
                $name = $record['name'] != '' ? join(Array($record['name'],'.',$zonename)) : $zonename;
				$record['content'] = str_replace("[zonename]", $zonename, $record['content']);
                $zone->addRecord($name, $record['type'], $record['content']);
            }
```

Thanx for the good work.

**Comment by tuxis-ie** [comment]
Thanks. Can you turn this into a PR?

**Comment by webvanced** [comment]
Hi, created the pull request.

**Comment by tuxis-ie** [comment]
Merged #153
